### ISSUE TITLE
unable to debug worker threads

### ISSUE THREAD

**Issue by dnalborczyk** [GoogleChromeLabs/ndb#277]
I'm unable to debug **worker threads**, ref: https://nodejs.org/dist/latest-v12.x/docs/api/worker_threads.html

are there any steps I'm missing to debug a worker thread?

### Steps to reproduce

**Tell us about your environment:**

* ndb version: **v1.0.52**
* Platform / OS version: **macOS Mojave 10.14.6 Beta**
* Node.js version: **v12.5.0**

**What steps will reproduce the problem?**

_Please include code that reproduces the issue._

```js
// index.js
const Worker = require('worker_threads');

const worker = new Worker('./script.js', { data: ... });
```

```js
// script.js
debugger // not being hit

console.log('foo')

```

**Comment by ak239** [comment]
The feature is not implemented yet but it is coming - should be available by the end of this week.

**Comment by dnalborczyk** [comment]
awesome, thanks for the update @ak239 !!

**Comment by ak239** [comment]
Support for workers available in `ndb@1.1.0` so ndb is the only one node debugger in a wild that supports both - child processes and workers. At the same time workers are not 100% ready - they requires node fix - https://github.com/nodejs/node/issues/28528 - should be available soon.

**Comment by dnalborczyk** [comment]
nice!!
